### PR TITLE
Fix some code comments style

### PR DESCRIPTION
**Pull request by xlwh**
Changelog category (leave one):
- Not for changelog (changelog entry is not required)

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Fix some code comments style. 

> Information about CI checks: https://clickhouse.tech/docs/en/development/continuous-integration/

### CODE REVIEW

File: src/TableFunctions/TableFunctionS3Cluster.h [hunk 1]
```diff
@@ -13,7 +13,7 @@ namespace DB
 class Context;
 
 /**
- * s3Cluster(cluster_name, source, [access_key_id, secret_access_key,] format, structure)
+ * S3 cluster(cluster_name, source, [access_key_id, secret_access_key,] format, structure)
```
Comment (kssenii) [inline src/TableFunctions/TableFunctionS3Cluster.h:16]: Previous version was correct, this one incorrect

Comment (xlwh) [reply src/TableFunctions/TableFunctionS3Cluster.h]: OK, thanks.

File: src/Storages/HDFS/StorageHDFS.h [hunk 2]
```diff
@@ -150,7 +150,7 @@ class HDFSSource : public SourceWithProgress, WithContext
     std::unique_ptr<ReadBuffer> read_buf;
     std::unique_ptr<QueryPipeline> pipeline;
     std::unique_ptr<PullingPipelineExecutor> reader;
-    /// onCancel and generate can be called concurrently
+    /// OnCancel and generate can be called concurrently.
```
Comment (kssenii) [inline src/Storages/HDFS/StorageHDFS.h:153]:
```suggestion
    /// onCancel and generate can be called concurrently.
```

File: src/TableFunctions/TableFunctionS3Cluster.h [hunk 3]
```diff
@@ -13,7 +13,7 @@ namespace DB
 class Context;
 
 /**
- * s3Cluster(cluster_name, source, [access_key_id, secret_access_key,] format, structure)
+ * s3 cluster(cluster_name, source, [access_key_id, secret_access_key,] format, structure)
```
Comment (kssenii) [inline src/TableFunctions/TableFunctionS3Cluster.h:16]:
```suggestion
 * s3Cluster(cluster_name, source, [access_key_id, secret_access_key,] format, structure)
```

**Review by kssenii — APPROVED** [review]

**Pull request merged**